roles/network_plugin/cilium/templates/cilium-config.yml.j2

```diff
@@ -61,7 +61,7 @@ data:
   # If you want cilium monitor to aggregate tracing for packets, set this level
   # to "low", "medium", or "maximum". The higher the level, the less packets
   # that will be seen in monitor output.
-  monitor-aggregation-level: "none"
+  monitor-aggregation: "none"
```

Comment:
Would you consider making this option configurable? With cilium v1.6, this value defaults to "medium".

Reply:
Sure, I will make it configurable! Should the default still be "none"?

Reply:
I think that it should be set to "medium" (see https://github.com/cilium/cilium/pull/8115).

Reply:
Alright, added the `cilium_monitor_aggregation` variable that defaults to "medium" now!